### DISCUSSION TITLE
specific transient forcings for AWI-CM3

### DISCUSSION THREAD

**Discussion by jian-shi72** [esm-tools/esm_tools#1031] [Models]
**Is your feature request related to a problem? Please describe.**
When I attempt to do a accelerated orbital transient experiment using AWI-CM3, I didn't find the switch that uses a list of orbital parameters/GHGs in a table (like "use_transient_forcing = true" for AWI-ESM2). Is there such a function for AWI-CM3 (especially for the orbit)?

**Describe the solution you'd like**
Just like the solution in AWI-ESM2:
use_transient_forcing: True
transient_forcing_table: /some/path/to/a/table

**Comment by nwieters** [comment]
Hi @jian-shi72,

I think this is a direct question to a specific model configuration. Therefore, I will convert this into a discussion for now.
We can change it back to an issue later, should it still be necessary.

**Comment by nwieters** [comment]
Hi @jian-shi72,

the variable `use_transient_forcing` is specific to the `echam` model.

@JanStreffing, do you know if AWICM3 has a similar parameter?

**Comment by JanStreffing** [comment]
I understand correctly that this is a method to speed up the change from one set of orbital parameters to another compared to how it would be in real time, while also not making a sharp jump?

**Comment by JanStreffing** [comment]
What we have for AWI-CM3 is transient orbital forcing, which just changes with the simulation year. It works for within +-1 millions years of the present. This would be `ORBMODE=variable_year` under: https://awi-cm3-documentation.readthedocs.io/en/latest/how_to.html#control-orbital-parameters 

There is no way to accelerate this by reading values from a list. We could of course build something like this in esm_tools / as a python script.

**Comment by JanStreffing** [comment]
Another alternative would be to modify the OpenIFS dates, so that the atmosphere actually jumps forward e.g. by 99 years after every 1 year of simulation upon restart. However we would need a small additional FESOM2 preprocessing script that does the same for FESOM2. As otherwise the years would be out of sync and leap years would cause trouble.

**Comment by jian-shi72** [comment]
Hi @JanStreffing, you're right. By reading a list of orbit forcing at a 10-years interval between two simulation years, the orbital scale simulation can run 10 times faster. I think it's better to mimic the `use_transient_forcing ` for echam instead of changing the OpenIFS dates so that one can easily accelerate their simulation at different rates.

**Comment by JanStreffing** [comment]
I see. Do you know where to start doing this? If not, let me know and we can brainstorm.

**Comment by jian-shi72** [comment]
I have ideas, doing some tests now.

获取Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
